lib/api/shared/src/main/scala/net/wiringbits/api/models/ForgotPassword.scala

```diff
@@ -12,8 +12,14 @@ object ForgotPassword {
   )
 
   @ApiModel(value = "ForgotPasswordResponse", description = "Response to the ForgotPasswordRequest")
-  case class Response(noData: String = "")
+  case class Response()
 
   implicit val forgotPasswordRequestFormat: Format[Request] = Json.format[Request]
-  implicit val forgotPasswordResponseFormat: Format[Response] = Json.format[Response]
+  //implicit val forgotPasswordResponseFormat: Format[Response] = Json.format[Response] 
+  implicit val forgotPasswordResponseFormat = OFormat[Response](Reads[Response] {
```

Comment:
Let's try creating a reusable function for defining these codecs to avoid repeating the same pattern everywhere